src/app/models/user.ts

```diff
@@ -144,6 +145,11 @@ export default (database: Sequelize): UserModel => {
         type: DataTypes.BOOLEAN,
         defaultValue: false,
       },
+      lastPasswordChangedAt: {
+        type: DataTypes.DATE,
+        defaultValue: new Date(),
```

Comment:
In this situation, on user creation, the Date will be set. Why not disallow null, and let on user creation the field have the value of the user creation? The first time will not invalidate something as this date will be always before any token renewal.

Reply:
Okay, that case could be logical to take into count. We can work with null values if the other solution is too complex or hard to understand.